bin/docker-build.sh

```diff
@@ -29,9 +30,16 @@ RUN echo "Deployment: $BLACK_KNIGHT_DEPLOYMENT" >> /app/public/round-table.txt
 RUN tar xvf /tmp/config.tar -C /
 EOF
 
-echo Building Container
-docker build -t "$repo:$new_tag" .
-docker rmi "$repo:$old_tag"
+echo "Black-Knight: I am Upgraded To Support Buiding Multi-Arch Docker Images"
+export DOCKER_CLI_EXPERIMENTAL=enabled
+echo "Black-Knight: I will be calling you "
+docker buildx create --use multi-arch-build
+echo "for this session "
+echo "Black-Knight: I am warming up "
+docker run --rm --privileged multiarch/qemu-user-static --reset -p yes
+echo "Black-Knight: Done warming up "
+echo "Black-Knight: I am building Docker Image for platform $target_platform"
```

Comment:
Unable to understand why all this is needed. 
Will the next line alone not be sufficient?

Reply:
@himani39 
line no 39 is required to test if black-knight will be supporting this build.
line 41 is added to get the platform value which is defined in black-knight
and other echo commands won't do much, i just added them to be more interactive and welcoming  logs